src/lib/get-codec-for-cid.ts

```diff
@@ -0,0 +1,159 @@
+import { CID } from 'multiformats'
+import multicodecs from 'multicodec'
+import { convert } from 'blockcodec-to-ipld-format'
```

Comment:
attempt to remove this dep.. its got a bunch of old deps that I would love to be rid of.

maybe patch-package for now if it really is needed? (things are working great now and I haven't tried removing this dep since first adding it in)